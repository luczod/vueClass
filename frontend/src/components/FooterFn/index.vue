<template>
  <footer>
    <h3>Party Time &copy; 2023</h3>
    <p>O melhor local para compartilhar seus eventos!</p>
  </footer>
</template>

<style scoped>
footer {
  background-color: #25282e;
  height: 10rem;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
}

footer h3 {
  color: #fff;
  margin-bottom: 1rem;
  font-weight: 300;
  font-size: 1.6rem;
}

footer p {
  color: #c1b696;
}
</style>

<script lang="js">
export default {
  name: 'FooterFn',
};
</script>
