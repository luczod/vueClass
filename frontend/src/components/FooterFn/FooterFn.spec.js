import { describe, expect, it } from 'vitest';
import { mount } from '@vue/test-utils';
import FooterFn from './index.vue';

describe('<FooterFn />', () => {
  it('should be defined', () => {
    const wrapper = mount(FooterFn);
    expect(wrapper).toBeDefined();
  });

  it('should match rendered', () => {
    const wrapper = mount(FooterFn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
