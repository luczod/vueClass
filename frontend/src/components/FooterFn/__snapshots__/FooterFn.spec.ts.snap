// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<FooterFn /> > should match rendered 1`] = `
<footer
  data-v-20a442e6=""
>
  <h3
    data-v-20a442e6=""
  >
    Party Time Â© 2023
  </h3>
  <p
    data-v-20a442e6=""
  >
    O melhor local para compartilhar seus eventos!
  </p>
</footer>
`;
